### PR TITLE
content: precompiling bits/stdc++.h

### PR DESCRIPTION
**Pull request by dongliu0426**

### CODE REVIEW

File: content/1_General/Running_Code_Locally.mdx [hunk 1]
```diff
@@ -274,6 +274,50 @@ work with the latter two. This is one of the reasons why you should **not** use
...
+cd bits
+g++ stdc++.h
+```
+
+Then, change `#include <bits/stdc++.h>` to `#include "bits/stdc++.h"`. Changing this tells the compiler to look for the local version.
+
+#### Option 2
+
+```
+g++ /usr/include/path_to/bits/stdc++.h
```
Comment (nwatx) [inline content/1_General/Running_Code_Locally.mdx:305]: For me, I had to use this with flag -H in order to get it to work, not sure about others.

File: content/1_General/Running_Code_Locally.mdx [hunk 2]
```diff
@@ -274,6 +274,50 @@ work with the latter two. This is one of the reasons why you should **not** use
...
+```
+Using this, you don't have to change `#include <bits/stdc++.h>` to `#include "bits/stdc++.h"`.
+
```
Comment (nwatx) [inline content/1_General/Running_Code_Locally.mdx:308]:
````suggestion

## Option 2 (Linux)

Tested on Ubuntu 20.04. Make sure to change `-std` header as needed.

```
cd /usr/include/x86_64-linux-gnu/bits
sudo g++ -H -std=c++17 -O2 stdc++.h
```
````

Comment (nwatx) [reply content/1_General/Running_Code_Locally.mdx:308]: if you want

**Review by nwatx — APPROVED** [review]

File: content/1_General/Running_Code_Locally.mdx [hunk 3]
```diff
@@ -274,6 +274,58 @@ work with the latter two. This is one of the reasons why you should **not** use
...
+## Option 2 (Linux)
+
+Tested on Ubuntu 20.04. Make sure to change `-std` header as needed.
+
+```
+cd /usr/include/x86_64-linux-gnu/c++/9/bits
+sudo g++ -H -std=c++17 -O2 stdc++.h
+```
+#### `-Winvalid-pch`
```
Comment (nwatx) [inline content/1_General/Running_Code_Locally.mdx:317]:
```suggestion

#### `-Winvalid-pch`
```

**Review by thecodingwizard — APPROVED** [review]

**Review by bqi343 — APPROVED** [review]

**Review by bqi343 — CHANGES_REQUESTED** [review]
actually idg why there are two option 2s

**Comment by dongliu0426** [comment]
ok i removed the second option 2 and added notes

**Review by bqi343 — APPROVED** [review]

**Pull request merged**